### PR TITLE
community/gitea: upgrade to 1.9.4

### PR DESCRIPTION
**Pull request by 6543**
# 1.8.3 -> 1.9.4

### CODE REVIEW

File: community/gitea/APKBUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Carlo Landmeter <clandmeter@gmail.com>
 pkgname=gitea
-pkgver=1.8.3
+pkgver=1.9.2
 pkgrel=1
```
Comment (maxice8) [inline community/gitea/APKBUILD]: pkgrel needs to reset to 0

**Comment by maxice8** [comment]
commit title should be

`<repo>/<pkg>: upgrade to <pkgver>`

File: community/gitea/APKBUILD (hunk 1 above)
Comment (6543) [reply community/gitea/APKBUILD]: thx

**Comment by 6543** [comment]
-> drone is down!!
`edge-x86_64: Error response from daemon: Failed to Setup IP tables: Unable to enable NAT rule: (iptables failed: iptables --wait -t nat -I POSTROUTING -s 192.168.240.0/20 ! -o br-85578df738e6 -j MASQUERADE: (fork/exec /sbin/iptables: no space left on device))`

**Comment by 6543** [comment]
ready to merge :rocket:

**Comment by 6543** [comment]
@maxice8 while I'm waiting for a upgrade ... a new version arived so I'll update the PR

-> this shuld take a new review
would you do this?

**Comment by 6543** [comment]
@maxice8 while I'm waiting for a upgrade ... a new version arived so I'll update the PR

-> this shuld take a new review
would you do this?

**Comment by 6543** [comment]
now tested with a new alpine container and with setup ... works fine!

**Comment by 6543** [comment]
@maxice8 is there something wich is blocking this PR?

**Comment by 6543** [comment]
@maxice8 is something blocking thebmerge?

I realy like tu use the new version on my instances?

**Comment by 6543** [comment]
@clandmeter you are the maintainer of this or?

**Comment by 6543** [comment]
hey next version is already coming out how long are you waiting?

**Comment by 6543** [comment]
@andypost you where reacently active at PR can you look at this please
nobodey is reacting (complane issues or aprove&merge it) :(

**Comment by andypost** [comment]
@6543 As I see you did not squashed commits, please use `git rebase -i HEAD~12` to squash them into one

**Comment by andypost** [comment]
@6543 also use `git commit --amend` to change commit message to `community/gitea: upgrade to 1.9.3`

**Comment by 6543** [comment]
o nice i stuck with a issue that wont be fixed ... https://github.com/go-gitea/gitea/issues/8126

**Comment by 6543** [comment]
@andypost but thanks for your quick and accurate answers!

**Comment by 6543** [comment]
Hey I'm going to **close** this vor now and wait until  gitea v1.10.x is out!
then I'll make a new PR wich is hopefuly merged befor the next release is already out again :angry: 

Everybody who is looking for a step in between, you can use my self created gitea package. (1.9.3)
-> no waranty
-> ask if you have futhure questions
-> download: https://cloud.obermui.de/s/GaccsQW5AYBCjcB/download

**Comment by 6543** [comment]
need a fix on the gitea site ... so this will upgrade to 1.9.4 instead of ...3

**Comment by 6543** [comment]
-> https://github.com/go-gitea/gitea/pull/8389

**Comment by Cogitri** [comment]
A bit confused what's the status of this right now. Also, sorry for the wait, Maxice8's notebook broke down.

**Comment by 6543** [comment]
@Cogitri Sory for him ... that's why

1. This PR worked but after quite some time dependencys had some changes because of the switch from go1.12 to 1.13 - avter it the buld process **was brocken** for gitea v1.9.x.

2. Diden't notice the brocken state until at this PR some devs react again (the build failures yesterday)

### 3. Current state:
Wait for new Release 1.9.4 witch fix gitea build process for v1.9.x

**Comment by Cogitri** [comment]
Alrighty 👍

**Comment by 6543** [comment]
@andypost gitea v1.9.4 is out :D

**Comment by 6543** [comment]
@Cogitri 
# State: ready

**Comment by 6543** [comment]
drone has an selfrelated error
`rm: can't remove '/home/buildozer/aports/community/gitea/src/pkg/mod/golang.org/x/crypto@v0.0.0-20190829043050-9756ffdc2472/bn256/bn256_test.go': Permission denied`

**Comment by 6543** [comment]
ok

**Comment by 6543** [comment]
@andypost is there a way to tell the CI to build this with go 1.12 not 1.13?

**Comment by andypost** [comment]
> @andypost is there a way to tell the CI to build this with go 1.12 not 1.13?
No, current edge version is https://pkgs.alpinelinux.org/packages?name=go&branch=edge

**Comment by 6543** [comment]
Ok I'll create a patch

**Comment by 6543** [comment]
dont look at the mess comits (i try to creat a patch wich should fix the ci issues ... but the resulution was to bump version 1.9.3 to 1.9.4  -> forgot it :( )

Now it should be ready!

any complains ?

**Comment by 6543** [comment]
@andypost  @Cogitri @clandmeter 

pleace review!

**Comment by 6543** [comment]
do I have to send this to 
* https://git.alpinelinux.org/aports/
* or https://gitlab.alpinelinux.org/alpine/aports

?

**Comment by Cogitri** [comment]
No, I was just on a trip during weekend, so didn't have time to review it. Sorry for the wait.

**Pull request closed without merging**